### PR TITLE
IO1-3571: changed faucet api, updated yml for claim and faucet

### PR DESCRIPTION
**Pull request by axelrod-blox**

### CODE REVIEW

File: .github/workflows/build_deploy.yml [hunk 1]
```diff
@@ -136,3 +136,59 @@ jobs:
...
+        if: github.ref == 'refs/heads/main'
+        run: >
+          SSV_NETWORKS="${{ env.STAGE_SSV_NETWORKS }}"
+          GOOGLE_TAG_SECRET="$GOOGLE_TAG_SECRET"
+          EXPLORER_URL="$EXPLORER_URL" BLOCKNATIVE_KEY="$BLOCKNATIVE_KEY" yarn build
+        env:
+          REACT_APP_FAUCET_PAGE: "true"
+          SSV_NETWORKS: ${{ env.STAGE_SSV_NETWORKS }}
+          BLOCKNATIVE_KEY: ${{ secrets.BLOCKNATIVE_KEY }}
+          EXPLORER_URL: "https://explorer.ssv.network/"
```
Comment (meshin-dev) [inline .github/workflows/build_deploy.yml:151]: You have `ssv_networks.explorerUrl`

File: .github/workflows/build_deploy.yml [hunk 2]
```diff
@@ -136,3 +136,59 @@ jobs:
...
+          LINK_SSV_DEV_DOCS: ${{ secrets.LINK_SSV_DEV_DOCS }}
+          REACT_APP_CAPTCHA_KEY: ${{ secrets.CAPTCHA_KEY_STAGE }}
+      - name: Deploy prod faucet
```
Comment (meshin-dev) [inline .github/workflows/build_deploy.yml:154]: New line

File: .github/workflows/build_deploy.yml [hunk 3]
```diff
@@ -136,3 +136,59 @@ jobs:
...
+          SSV_NETWORKS="${{ env.STAGE_SSV_NETWORKS }}"
+          GOOGLE_TAG_SECRET="$GOOGLE_TAG_SECRET"
+          EXPLORER_URL="$EXPLORER_URL" BLOCKNATIVE_KEY="$BLOCKNATIVE_KEY" PROD_CLAIM_PAGE="$PROD_CLAIM_PAGE" yarn build
+        env:
+          PROD_CLAIM_PAGE: "true"
+          BLOX_INFRA_API: 'https://api.bloxstaking.com'
+          BLOCKNATIVE_KEY: ${{ secrets.BLOCKNATIVE_KEY }}
+          EXPLORER_URL: "https://explorer.shifu.ssv.network"
+          LINK_SSV_DEV_DOCS: ${{ secrets.LINK_SSV_DEV_DOCS }}
+          GOOGLE_TAG_SECRET: ${{ secrets.PROD_GOOGLE_TAG_SECRET }}
```
Comment (meshin-dev) [inline .github/workflows/build_deploy.yml:180]: You have `ssv_networks.explorerUrl`
You have `ssv_networks.googleTagSecret `

**Review by meshin-dev — CHANGES_REQUESTED** [review]

File: .github/workflows/build_deploy.yml (hunk 2 above)
Comment (axelrod-blox) [reply .github/workflows/build_deploy.yml:154]: done

File: .github/workflows/build_deploy.yml (hunk 3 above)
Comment (axelrod-blox) [reply .github/workflows/build_deploy.yml:180]: done

File: .github/workflows/build_deploy.yml (hunk 1 above)
Comment (axelrod-blox) [reply .github/workflows/build_deploy.yml:151]: done

**Pull request merged**